test/blackbox-tests/test-cases/melange/aliases.t

```diff
@@ -67,3 +90,17 @@ If user defines an alias, the default alias is not used for that stanza
   Error: Alias "melange" specified on the command line is empty.
   It is not defined in . or any of its descendants.
   [1]
+
+Even if user defines an alias, dune default alias should still work
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
```

Comment:
Add `(emit_stdlib false)` to speed up the tests

Reply:
reflected in 7c49c1d7027915cc9aff8d3a76fa58081ba84ecd